src/components/implementation/nicmgr/dpdk/nicmgr.c

```diff
@@ -129,12 +153,32 @@ nic_send_packet(shm_bm_objid_t pktid, u16_t pkt_offset, u16_t pkt_len)
 	
 	buf.paddr   = data_paddr;
 	buf.pkt_len = pkt_len;
-
+#if 0
```

Comment:
What's the point of this check? Remove the code if we don't need or change the macro to something make sense.